client/www/scripts/vendor/nv.d3.js

```diff
@@ -855,11 +855,12 @@ window.nv.tooltip.* also has various helper methods.
               }
 
               setTimeout(function() {
-
                   while (purging.length) {
                      var removeMe = purging.pop();
+                  if (removeMe.parentNode) {
```

Comment:
The error isn't critical to this fix. I can pull this out so we can continue with this fix, then go about getting this fixed upstream in nvd3 separately.